src/pages/Login/Login.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from "react";
+
+import "./Login.scss";
+
+// import { Link } from "react-router-dom";
+
+class Login extends Component {
+    constructor() {
+        super();
+        this.state = {
+            idValue: "",
+            pwValue: "",
+            isBtnActive: false,
+        };
+    }
+
+    handleIdValue = (event) => {
+        this.setState(
+            {
+                idValue: event.target.value,
+            },
+            () => this.checkBtnState()
+        );
+    };
+
+    handlePwValue = (event) => {
+        this.setState(
+            {
+                pwValue: event.target.value,
+            },
+            () => this.checkBtnState()
+        );
+    };
+
+    checkBtnState = () => {
+        const { idValue, pwValue } = this.state;
+        const isActive = idValue.length > 5 && pwValue.length >= 8;
+
+        this.setState({
+            isBtnActive: isActive ? true : false,
```

Comment:
isBtnActive 값 자체가 boolean타입으로 받는 것이므로 true, false는 생략이 가능합니다.
```suggestion
            isBtnActive: isActive,
```